lib/resque_spec/matchers.rb

```diff
@@ -185,13 +201,13 @@ def schedule_queue_for(actual)
   chain :at do |timestamp|
     @interval = nil
     @time = timestamp
-    @time_info = "at #{@time}"
+    @time_info = " at #{@time}"
   end
 
   chain :in do |interval|
     @time = nil
     @interval = interval
-    @time_info = "in #{@interval} seconds"
+    @time_info = " in #{@interval} seconds"
```

Comment:
As an example, addressing point 2:

Line 117 could be:

"but got #{actual} with [#{actual_queue_args_str}]#{@times_info} instead"

Line 248  could be:

`["expected that #{actual} would not have [#{expected_args.join(', ')}] scheduled", "#{@time_info}", actual_queue_message].join(', ')

The ", " is inserted by the caller as needed. Each part of the message is just a building block and not specifying how it is joined to the rest of the message.

Does that make sense?